plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -257,6 +285,17 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
     locationPuckManager?.updateCurrentBearing(*bearing, options = options)
   }
 
+  /**
+   * Called whenever the accuracy radius is updated.
+   * @param radius - supports multiple radius value to create more complex animations with intermediate points.
+   *  Last [radius] value will always be the animator target for next animation.
+   * @param options - if specified explicitly will apply current animator option to radius animation.
+   *  Otherwise default animator options will be used.
+   */
+  override fun onAccuracyRadiusUpdated(vararg radius: Double, options: (ValueAnimator.() -> Unit)?) {
```

Comment:
IMO its a little bit strange to use `vararg` with optional parameter, especially as last parameter
you can't use it without calling by name

Reply:
This is algin with others like `onLocationUpdated` and `onBearingUpdated`. Can you create a ticket describing details about this issue and how to improve it? We can iterate on this in the future.